Mods/defaultTemplates/mod.json

```diff
@@ -2,13 +2,8 @@
 	"name" : "Default templates",
 	"description" : "A set of random map templates for VCMI",
 
-	"version" : "0.100",
+	"version" : "0.99a",

```

Comment:
Let's not use such versions because launcher converts them into numbers.
So it should be major.minor.patch[optional]